jina/executors/metas.py

```diff
@@ -144,7 +136,7 @@
 
     .. note::
 
-        ``separated_workspace``, ``pea_workspace`` and ``pea_id`` is set in a way that when the executor ``A`` is used as
+        ``pea_workspace`` and ``pea_id`` is set in a way that when the executor ``A`` is used as
```

Comment:
pea_workspace should as well be removed I think as now the workspace is uniquely identified

Reply:
good idea. Let's do this in a dedicated pr.